stories/internal/Export.tsx

```diff
@@ -0,0 +1,238 @@
+// (C) 2007-2018 GoodData Corporation
+import * as React from 'react';
+import { IExportConfig, IExportResponse } from '@gooddata/gooddata-js';
+import { screenshotWrap } from '@gooddata/test-storybook';
+import { action } from '@storybook/addon-actions';
+import { storiesOf } from '@storybook/react';
+
+import { BarChart, PivotTable, Table } from '../../src';
+import { Visualization } from '../../src/components/uri/Visualization';
+import { IExportFunction, OnExportReady } from '../../src/interfaces/Events';
+
+import { onErrorHandler } from '../mocks';
+import {
+    ATTRIBUTE_1,
+    ATTRIBUTE_2,
+    MEASURE_1,
+    MEASURE_2,
+    TOTAL_M1_A1,
+    TOTAL_M2_A1
+} from '../data/componentProps';
+
+const wrapperStyle = { width: 800, height: 400 };
+const buttonStyle = { marginRight: 10 };
+
+interface IWrapperComponentProps {
+    children: (onExportReady: OnExportReady) => React.ReactChild;
+}
+
+class WrapperComponent extends React.Component<IWrapperComponentProps> {
+    private exportResult: IExportFunction;
+
+    constructor(props: IWrapperComponentProps) {
+        super(props);
+        this.onExportReady = this.onExportReady.bind(this);
+        this.exportToCSV = this.exportToCSV.bind(this);
+        this.exportToXLSX = this.exportToXLSX.bind(this);
+        this.exportWithCustomName = this.exportWithCustomName.bind(this);
+        this.exportWithMergeHeaders = this.exportWithMergeHeaders.bind(this);
+        this.doExport = this.doExport.bind(this);
+    }
+
+    public render() {
+        return (
+            <div style={wrapperStyle}>
+                {this.props.children(this.onExportReady)}
+                <br />
+                <div>
+                    <button style={buttonStyle} onClick={this.exportToCSV}>Export CSV</button>
+                    <button style={buttonStyle} onClick={this.exportToXLSX}>Export XLSX</button>
+                    <button style={buttonStyle} onClick={this.exportWithCustomName}>
+                        Export with custom name "CustomName"
```

Comment:
We have 3 buttons here, but I cannot see them in storybook images.
Can you please explain?

Reply:
The buttons are placed out of view-port. I will update the images again

Reply:
updated